### PR TITLE
Branch created not complete just yet though

### PR DESCRIPTION
**Pull request by seenaomi**
Added gwc-5.jpg image to the img/girls dir

edited the style.css sheet
- specifically wrap-left and img.wrap-left classes 
- added about-title class and about-post class to the style.css sheet

Added text, image, and adjusted layout of about page

### CODE REVIEW

**Comment by seenaomi** [comment]
@sandikbarr @Gmfholley can you please take a good look over the text? The last paragraph specifically might need another editors eye. Any feedback on layout etc is greatly appreciated as well. I haven't added the FAQ direct link as Im not sure if it's done yet. Thank you! I respect both of your opinions and won't cry. :)

**Comment by seenaomi** [comment]
I found the title of the FAQ page which yknow was faq.html, lol. So that's been corrected.

File: css/style.css [hunk 1]
```diff
@@ -142,18 +142,35 @@ p > a {
   margin-left: 10px;
 }
 
+/*I'm only seeing wrap-left being used on the About page but it's listed under .team
```
Comment (sandikbarr) [inline css/style.css:145]: This seems good. We could probably clean up our .team css and stuff. For example, looks like I've got the team class in volunteers still, but not sure it's really using it. But things are looking good for what we've got, so I'm good with it for getting the site out there.

**Comment by sandikbarr** [comment]
Looks great! Ready to merge 👍

**Comment by seenaomi** [comment]
Branch successfully merged.

**Pull request merged**